pca9685:
  ros__parameters:
    device: "/dev/i2c-1"
    address: 64           # 0x40 in hexadecimal is 64 in decimal
    frequency: 333
    timeout: [5000, 5000, 5000, 5000, 5000, 5000, 5000, 5000,
              5000, 5000, 5000, 5000, 5000, 5000, 5000, 5000]
    pwm_min: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
    pwm_max: [65535, 65535, 65535, 65535, 65535, 65535, 65535, 65535,
              65535, 65535, 65535, 65535, 65535, 65535, 65535, 65535]
    timeout_value: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]

ctrl:
  ros__parameters:
    freq: 30
    angle_range: [-60, 60]
    pwm_range: [21500, 51500]
    start_wing: [1.0, 30.0, 0.0]
    start_flap: [30.0, 0.0]
    clamp_flap: [-40, 40]
    
